content/400-reference/200-api-reference/050-prisma-client-reference.mdx

```diff
@@ -5429,7 +5429,17 @@ const getUsers = await prisma.user.findMany({
 
 ## Client methods
 
-Client-level methods are prefixed by `$`.
+### Remarks
+
+- Client-level methods are prefixed by `$`.
```

Comment:
Is this really a "Remark" and not the fundamental description of what Client methods are? (Which for me would mean it should be places after the `##` and not under `### Remarks`.)

Reply:
I'll move the note